### DISCUSSION TITLE
how to upload  already exist  asset to remote cache server

### DISCUSSION THREAD

**Discussion by nocanstillbb** [microsoft/vcpkg#35224] [General]
in linux , binaries cached in  `~/.cache/vcpkg/archives/`  
I can upload them direct.  but where  asset caching to ?
files in `vcpkg_root/downloads` 's   name  is't hash string,  so i think they are not

**Comment by autoantwort** [comment]
You can not upload already downloaded assets to the asset cache server (maybe manually). If you have configured an asset cache you can download the files again (e.g. by deleting the downloads folder), then the files are uploaded to the asset cache.

**Comment by nocanstillbb** [comment]
ok ,。。。
